migrate/migrations/000010_add-feeds-table.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE feeds (
+    id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+
+    post_type TINYINT UNSIGNED,
+    audience TINYINT UNSIGNED,
```

Comment:
Similar idea for above - we might want to be able to filter feeds on whether or not they're private/public